src/level/robotmain.cpp

```diff
@@ -5839,6 +5839,17 @@ void CRobotMain::QuickLoad()
     m_playerProfile->LoadScene(dir);
 }
 
+void CRobotMain::LoadSaveFromDirName(const std::string& gameDir)
+{
+    std::string dir = m_playerProfile->GetSaveFile(gameDir);
+    if(!CResourceManager::Exists(dir))
+    {
+        GetLogger()->Error("Save slot not found\n");
+        return;
+    }
+    m_playerProfile->LoadScene(dir);
```

Comment:
I think this will have the same issue as #812.

We could add a boolean parameter to ```CPlayerProfile::LoadScene()``` to handle these two special cases.

Maybe ```LoadScene(std::string dir, bool loadFinishedFromCat = false);```

Reply:
Ouch. Issue confirmed...

Reply:
It looks that might work, maybe that's the proper fix for #812 too.

Reply:
That seems like it worked, at least for me.